arm-compute/quickstart-templates/aliases.json

```diff
@@ -16,34 +16,69 @@
             "version": "latest",
             "architecture": "x64"
           },
+          "CentOS85Gen2":  {
+            "publisher":  "OpenLogic",
+            "offer":  "CentOS",
+            "sku":  "8_5-gen2",
+            "version":  "latest",
+            "architecture": "x64"
```

Comment:
@kangsun-ctrl Could you please help confirm whether these newly added images still support `x64` architecture?